src/common/openshiftKinds.js

```diff
@@ -0,0 +1,62 @@
+const namespaceRequestDef = {
```

Comment:
Not sure what is meant by Kinds?  Any thought on a different name ... maybe openshiftHelpers.js?

Reply:
I can move it to helpers, but wanted to have k8s/openshift types in a separate file, as they are mostly simple objects